tests/engines/registries/acceptance/overview/form-schemas-test.ts

```diff
@@ -14,7 +14,10 @@ module('Registries | Acceptance | overview form schemas', hooks => {
         server.loadFixtures('registration-schemas');
 
         for (const registrationSchema of server.schema.registrationSchemas.all().models) {
-            const registration = server.create('registration', { registrationSchema });
+            const registration = server.create('registration', {
+                registrationSchema,
+                provider: server.create('registration-provider', { id: 'osf' }),
```

Comment:
Check if this is needed. If the provider is set to OSF as default, do we need to restate that here?